PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD001.xml

```diff
@@ -144,12 +144,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                                <desc>Regular hand, The manuscript was copied by two scribe.
+                                <desc>Regular hand. The manuscript was copied by two scribe.
                                 </desc>
                                 <seg type="ink">Black, red.</seg>
                                 <date from="1950" to="1958" evidence="internal-date"> </date>
-                                <note>Marigetā Čakola (Marigetā Damḍa's maternal grandfather) copied the first 105 folios, 
-                                    while Marigetā Damḍa copied the last 21, with a twelve-year gap between the two productions.</note>
+                                <note>Marigetā Čakola (<persName ref="PRS14417MarigetaDamseEndalew"/> maternal grandfather)
```

Comment:
`(maternal grandfather of <persName ref="PRS14417MarigetaDamseEndalew"/>)`

Can you provide a PRS ID for Marigetā Čakola, as well?